.github/workflows/tail-call.yml

```
@@ -2,12 +2,14 @@ name: Tail calling interpreter
 on:
   pull_request:
     paths:
+      - '.github/workflows/tail-call.yml'      
```

This feels wrong to me, but if it's needed, then sure. Hopefully someone else can sign off on this part of the change.

---

This is fine: when changing this file, it's good to run it as well to make sure we didn't break it -- which is rather too easy with YAML.

---

(And let's strip the trailing spaces.)

```suggestion
      - '.github/workflows/tail-call.yml'
```

(We should add `yaml` to this list, feel free to include it in the PR if you like:) https://github.com/python/cpython/blob/2904ec2273762df58645a8e245b2281884855b8c/.pre-commit-config.yaml#L49